SpinCore_pp/config_parser_fn.py

```diff
@@ -157,6 +151,7 @@ class configuration(object):
             None,
             "number of IR experiments collected in the ODNP experiment",
         ),
+        # reviewed to here
```

Comment:
next time, come back and review descriptions/parameters from here

Reply:
think we did this